packages/sdk-api/src/api/services/ReceivableService.ts

```diff
@@ -144,7 +144,7 @@ export class ReceivableService extends CommonService {
           created_at__gte: createdAtGte,
           created_at__lte: createdAtLte,
           counterpart_name: counterpartName,
-          counterpart_name__contains: counterpartNameContains,
+          counterpart_id: counterpartId,
```

Comment:
Shouldn't we provide this new parameter in the `useReceivables` component?

https://github.com/team-monite/monite-sdk/blob/main/packages/sdk-react/src/components/receivables/InvoicesTable/InvoicesTable.tsx#L74

Reply:
Nope, it already used with [`currentFilters[FILTER_TYPE_CUSTOMER]`](https://github.com/team-monite/monite-sdk/blob/410eb6195cec3dfc541142af893ba3d3591bd8f0/packages/sdk-react/src/components/receivables/InvoicesTable/InvoicesTable.tsx#L92). API has been changed, as services are not generatable for us, there is no way to catch a type error.